injector/src/main/java/edu/ucr/cs/riple/injector/location/OnParameter.java

```diff
@@ -44,13 +46,17 @@ public class OnParameter extends Location {
   public final int index;
   private final SignatureMatcher matcher;
 
-  public OnParameter(String path, String clazz, String method, int index) {
+  public OnParameter(Path path, String clazz, String method, int index) {
     super(LocationType.PARAMETER, path, clazz);
     this.method = method;
     this.index = index;
     this.matcher = new SignatureMatcher(method);
   }
 
+  public OnParameter(String path, String clazz, String method, int index) {
```

Comment:
Same question as above, and similar for `OnField`, I suppose.

Reply:
Please [here](https://github.com/ucr-riple/NullAwayAnnotator/pull/121#discussion_r1059974805).